docs/genai/howto/install.md

```diff
@@ -23,61 +27,120 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
 
 Ensure that the `CUDA_PATH` environment variable is set to the location of your CUDA installation.
 
+Versions later than ONNX Runtime release bundle CUDA 12 by default. CUDA 11 is also supported in a separate package. Instructions are given below.
+
 ## Python packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these set of packages (CPU, DirectML, CUDA) should be installed in your environment.
 
 ### CPU
 
+#### Intel CPU
+
 ```bash
-pip install numpy
+pip install onnxruntime
+pip install onnxruntime-genai --pre
+```
+
+#### Arm CPU
+
+```bash
+pip install onnxruntime-qnn
 pip install onnxruntime-genai --pre
 ```
 
-### DirectML
 
-Append `-directml` for the library that is optimized for DirectML on Windows
+### DirectML
 
 ```bash
-pip install numpy
+pip install onnxruntime-directml
 pip install onnxruntime-genai-directml --pre
 ```
 
 ### CUDA
 
-Append `-cuda` for the library that is optimized for CUDA environments
-
 #### CUDA 11
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
+# Install ORT nightly CUDA 11, change this to released version when it is released
+pip install ort-nightly-gpu --index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-cuda-11-nightly/pypi/simple
+
+
+# Install onnxruntime-genai built for CUDA 11
+pip install onnxruntime-genai-cuda-11 --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
 ```
 
 #### CUDA 12
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+# Install ORT nightly CUDA 12, change this to released version when it is released
+pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+
+# Install onnxruntime-genai build for CUDA 12
+pip install onnxruntime-genai-cuda --pre --index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
```

Comment:
we can publish onnxruntime-genai-cuda to pypi too after sperating the onnxruntime.dll from genai

Reply:
Fixed